unlist-posts.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name:     Unlist Posts & Pages
  * Plugin URI:      https://github.com/Nikschavan/hide-post
- * Description:     Unlist Posts from dispying anywhere on the site, only access the post with a direct link.
+ * Description:     With this plugin you can hide post, pages and custom items from your site and make them only accessible with the direct link.
```

Comment:
```suggestion
 * Description:     Hide posts, pages or  custom items from your site and make them accessible only with the direct link.
```

A suggestion in the description change